python/cudf/cudf/_lib/utils.pyx

```diff
@@ -37,6 +37,30 @@ PARQUET_META_TYPE_MAP = {
 }
 
 
+cdef table_view table_view_from_columns(columns) except*:
+    """Create a cudf::table_view from an iterable of Columns."""
+    cdef vector[column_view] column_views
+
+    cdef Column col
+    for col in columns:
+        column_views.push_back(col.view())
+
+    return table_view(column_views)
+
+
+cdef table_view table_view_from_table(tbl, ignore_index=False) except*:
```

Comment:
I think the right name of this is `table_view_from_frame`. Eventually we want `table_view_from_data`.

Reply:
You're half right. A table view is constructed from a sequence of columns, not a mapping, so we will never want `table_view_from_data`. We will simply use the already existing function `table_view_from_columns` and delete this function. Since the plan is to delete this function at that point, would you mind be being lazy and not changing the function name here :)